extensions/amp-lightbox-viewer/0.1/amp-lightbox-viewer.js

```diff
@@ -133,16 +138,33 @@ export class AmpLightboxViewer extends AMP.BaseElement {
             const deepClone = !element.classList.contains(
                 'i-amphtml-element');
             const nodeToClone = element.cloneNode(deepClone);
+            nodeToClone.removeAttribute('on');
+            const descText = this.manager_.getDescription(element);
+            if (descText) {
+              nodeToClone.descriptionText = descText;
```

Comment:
please put a TODO to store this and the lightboxid in a data structure other than the node itself.